src/lib/startingProcess.js

```diff
@@ -12,12 +12,12 @@ const startingProcess = (startFn, stopFn) => {
   Logger.verbose(`starting ${processName}...`, { startTime })
 
   process.on('uncaughtExceptionMonitor', (err) => {
-    Logger.error(`uncaughtExceptionMonitor in ${processName}: ${err?.message}`, { err })
+    Logger.error(`uncaughtExceptionMonitor in ${processName}: ${err?.stack}`, { err, stack: err?.stack })
```

Comment:
Can we leave logging message in the first param to Logger, and add stack to the second one:
```
Logger.error(`uncaughtExceptionMonitor in ${processName}: ${err?.message}`, { stack: err?.stack })
```

Reply:
I tried that first, but then the stack is not shown in the console.

Reply:
strange...